src/components/TextField/TextField.css

```diff
@@ -0,0 +1,65 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+@import "src/css/variables/fonts.css";
+
+// work is in progress to set styling/theme options for all Fabric label instances
+.y-text-field .ms-Label {
+  @extend text__mediumSub;
+  font-weight: $fontWeight__bold;
+  color: $textColor__secondary;
+  padding-bottom: 4px;
+}
+
+.y-text-field .ms-TextField-fieldGroup {
+  line-height: 3.3rem;
+  height: 36px;
+}
+
+.y-text-field.ms-TextField--multiline .ms-TextField-field {
```

Comment:
can we remove the multiline code from Textfield now that the components are being separated?

Reply:
yup, removed.